### PR TITLE
feat(@angular-devkit/core): make new workspace API public

### PR DESCRIPTION
**Pull request by clydin**

### CODE REVIEW

File: packages/angular_devkit/core/src/_golden-api.ts [hunk 1]
```diff
@@ -51,3 +51,7 @@ export * from './utils/lang';
 export * from './virtual-fs/path';
 export * from './virtual-fs/host/index';
 // End virtualFs namespace
+
+// Start workspace namespace
+export * from './workspace';
```
Comment (alan-agius4) [inline packages/angular_devkit/core/src/_golden-api.ts]: This should be `workspace/index` are api guardian uses classic module resolution

Comment (clydin) [reply packages/angular_devkit/core/src/_golden-api.ts]: It passes though?

Comment (alan-agius4) [reply packages/angular_devkit/core/src/_golden-api.ts]: Yeah, cause API guardian completely ignores such exports/imports.

Comment (clydin) [reply packages/angular_devkit/core/src/_golden-api.ts]: Updated.

**Pull request merged**